### PR TITLE
WT-12769 Add eviction operator to the workload generator

### PR DESCRIPTION
**Pull request by pmacko86**

### CODE REVIEW

File: test/model/src/core/util.cpp [hunk 1]
```diff
@@ -304,6 +304,40 @@ parse_uint64(const char *str, const char **end)
...
+    WT_SESSION *session;
+    int ret = conn->open_session(conn, nullptr, nullptr, &session);
+    if (ret != 0)
+        throw wiredtiger_exception("Cannot open a session: ", ret);
+    wiredtiger_session_guard session_guard(session);
+
+    ret = session->begin_transaction(session, "ignore_prepare=true");
+    if (ret != 0)
+        throw wiredtiger_exception("Transaction begin failed: ", ret);
+    session_guard.rollback_on_close();
```
Comment (korteland) [inline test/model/src/core/util.cpp:323]: I think this `rollback_on_close` mechanism works fine, but I'm not sure why it's necessary. WiredTiger will rollback any running transactions when `session->close` is called.

File: test/model/src/driver/kv_workload_generator.cpp [hunk 2]
```diff
@@ -60,6 +60,8 @@ kv_workload_generator_spec::kv_workload_generator_spec()
 
     checkpoint = 0.02;
     crash = 0.002;
+    /* FIXME-WT-12972 Enable the eviction operator when it is safe to do so. Set to 0.1. */
```
Comment (korteland) [inline test/model/src/driver/kv_workload_generator.cpp:63]: Very cool that this is already finding (potential) bugs :-)

File: test/model/src/core/util.cpp (hunk 1 above)
Comment (pmacko86) [reply test/model/src/core/util.cpp]: Nice! I backed that out in [571b6f8](https://github.com/wiredtiger/wiredtiger/pull/10577/commits/571b6f8f95859ec3722deb0face1138b98555877).

**Review by korteland — APPROVED** [review]
LGTM!

**Comment by etienneptl** [comment]
I completely forgot about this one @pmacko86, my apologies. I will review it today.

**Review by etienneptl — APPROVED** [review]
My apologies for the extremely late review. LGTM!

**Pull request merged**